ecs-logs-collector.sh

```diff
@@ -392,8 +393,8 @@ get_ecs_init_logs() {
   fi
 
   mkdir -p ${dstdir}
-  for entry in ecs-init.log*; do
-    cp -fR /var/log/ecs/${entry} ${dstdir}/
+  for entry in $(ls /var/log/ecs/ecs-init.log* 1> /dev/null 2>&1); do
```

Comment:
This prevents the output from making its way to the loop, I think instead we'd want to make only the warning go to `/dev/null`:

```suggestion
  for entry in $(ls /var/log/ecs/ecs-init.log* 2>/dev/null); do
```